extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/providers/KnownOidcProviders.java

```diff
@@ -98,4 +100,24 @@ private static OidcTenantConfig apple() {
         ret.getCredentials().getJwt().setAudience("https://appleid.apple.com/");
         return ret;
     }
+
+    private static OidcTenantConfig spotify() {
+        // See https://developer.spotify.com/documentation/general/guides/authorization/code-flow/
+        OidcTenantConfig ret = new OidcTenantConfig();
+        ret.setDiscoveryEnabled(false);
+        ret.setAuthServerUrl("https://accounts.spotify.com/");
+        ret.setApplicationType(OidcTenantConfig.ApplicationType.WEB_APP);
+        ret.setAuthorizationPath("https://accounts.spotify.com/authorize");
```

Comment:
If you make this relative, it helps me override those URIs for OIDC mock tests :)
Same for next line.